setup.cfg

```diff
@@ -18,4 +18,20 @@ classifiers =
 
 [options]
 packages = twoaxistracking
-python_requires = >=3.6
\ No newline at end of file
+python_requires = >=3.7
+install_requires =
+    pvlib
+    numpy
+    pandas
+    matplotlib
+    shapely
```

Comment:
At some point (not this PR) we should figure out if any of these need a minimum version.

Reply:
How does one test that? Try different installations manually?

I believe that pandas is only used for tests, and only for extremely basic stuff, so I doubt that there will be any restrictions on that.

Reply:
Sure, it's reasonable to go back in time until either (1) your tests start failing or (2) you can't get the old dependency versions working on the python versions we support.  [NEP-29](https://numpy.org/neps/nep-0029-deprecation_policy.html) is also a good rule of thumb.